validate.swift

```diff
@@ -14,7 +14,7 @@ let timeoutIntervalForResource = 6000.0
 let rawURLComponentsBase = URLComponents(string: "https://raw.githubusercontent.com")!
 
 // master package list to compare against
-let masterPackageList = rawURLComponentsBase.url!.appendingPathComponent("SwiftPackageIndex/MasterPackageList/master/packages.json")
+let masterPackageList = rawURLComponentsBase.url!.appendingPathComponent("SwiftPackageIndex/PackageList/main/packages.json")
```

Comment:
aha! it was a very similar thing 😄

Reply:
yep - just did the same over in #410 :)

But I'll take yours. You got caught right in our update ;)